README.md

```diff
@@ -114,22 +114,21 @@ After you're finished please remove all the comments and instructions!
 
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-- [Live Demo Link](https://deploy-preview-31--rentbikes.netlify.app)
+- [Live Demo Link]
```

Comment:
- [ ] When I try to see the available bikes, it seems the styling gets in the way because the container is very tiny:

![image](https://user-images.githubusercontent.com/57421823/230703680-4fcfc488-6145-4c3e-a5b1-5697905bccf1.png)

Also, if I add another bike, it gets added to the same container, on top of the other one:

![image](https://user-images.githubusercontent.com/57421823/230703705-2a94f255-dd53-40d8-97ed-7c52aff3f278.png)

This happens both on the mobile and desktop views.  Kindly, correct the styling a little so it's possible to see the bikes available, like in the original design. 💪